vllm/_custom_ops.py

```diff
@@ -281,6 +281,45 @@ def fused_add_rms_norm(input: torch.Tensor, residual: torch.Tensor,
     torch.ops._C.fused_add_rms_norm(input, residual, weight, epsilon)
 
 
+def apply_repetition_penalties_torch(
```

Comment:
For example, simply put this under `@torch.compile`. Does it help match the CUDA performance?